engine/src/field.cpp

```diff
@@ -1493,6 +1493,89 @@ void MCField::formattabstops(Properties which, uint16_t *tabs, uint16_t tab_coun
     /* UNCHECKED */ MCListCopyAsString(*t_list, r_result);
 }
 
+bool MCField::parsetabalignments(MCStringRef p_data, intenum_t *&r_alignments, uindex_t &r_alignment_count)
```

Comment:
At the moment, `kMCParagraphTextAlignLeft` etc. are defined in [an anonymous enumeration in `paragraph.h`](https://github.com/livecodeian/livecode/blob/d7aaf4331aa5a1b59b05c49e5f659ee4382297c2/engine/src/paragraf.h#L76).  I suggest naming it (for example, `enum MCParagraphTextAlign`) and then using `MCParagraphTextAlign` instead of `intenum_t` where applicable throughout this patch.